xml/libvirt_configuration_virsh.xml

```diff
@@ -1135,6 +1135,38 @@ pci_0000_04_11_5</screen>
    </sect3>
   </sect2>
  </sect1>
+ <sect1 xml:id="libvirt-config-listing-host-devs">
+  <title>Listing Attached Devices</title>
+  <para>
+   Although there is no mechanism in &virsh; to list all &vmhost;'s devices
+   that have already been attached to its &vmguest;s, you can list all devices
+   attached to a specific &vmguest; by running the following command:
+  </para>
+<screen>virsh dumpxml <replaceable>VMGUEST_NAME</replaceable> | xpath /domain/devices/hostdev</screen>
```

Comment:
Does the xpath query need a '-e'? E.g. xpath -e /domain/devices/hostdev. At least it does for me on my SLES15 SP2 test host.